shaders/src/light_indirect.fs

```diff
@@ -209,17 +209,17 @@ vec3 importanceSamplingVNdfDggx(vec2 u, float roughness, vec3 v) {
     return h;
 }
 
-float prefilteredImportanceSampling(float ipdf, uint iblMaxMipLevel) {
+float prefilteredImportanceSampling(float ipdf, vec2 iblMaxMipLevel) {
     // See: "Real-time Shading with Filtered Importance Sampling", Jaroslav Krivanek
     // Prefiltering doesn't work with anisotropy
     const float numSamples = float(IBL_INTEGRATION_IMPORTANCE_SAMPLING_COUNT);
     const float invNumSamples = 1.0 / float(numSamples);
-    const float dim = float(1u << iblMaxMipLevel);
```

Comment:
Why didn't you just do a `uint(iblMaxMipLevel)` explicit conversion instead of adding an extra uniform?